sgkit/io/dataset.py

```diff
@@ -13,6 +13,7 @@ def save_dataset(
     ds: Dataset,
     store: Union[PathType, MutableMapping[str, bytes]],
     storage_options: Optional[Dict[str, str]] = None,
+    auto_rechunk: bool = False,
```

Comment:
Safer to make the default ``None`` here in case we want to do something clever in future?

Reply:
I also thought this - but checking the rest of the API `bool` options aren't following this pattern in sgkit.

Reply:
Changed.